serde/src/de/impls.rs

```diff
@@ -70,6 +103,32 @@ impl Deserialize for () {
 
 ///////////////////////////////////////////////////////////////////////////////
 
+#[cfg(all(not(feature = "std"), not(feature = "rustc_unicode")))]
```

Comment:
Yeah, but I'd prefer one of the stable Unicode modules like from https://github.com/unicode-rs so it could be used by stable rust. I'd like to avoid an implementation for stable rust and for nightly rust as much as possible.